Makefile

```diff
@@ -53,5 +53,12 @@ tests/GNTAllocation.bin: contracts/GNTAllocation.sol
 tests/GNTAllocation.abi: contracts/GNTAllocation.sol
 	solc --bin --abi --optimize contracts/GNTAllocation.sol | awk '/======= GNTAllocation =======/,/======= GolemNetworkToken =======/' | grep '\[.*\]' > tests/GNTAllocation.abi
 
+tests/Wallet.bin: contracts/Wallet.sol
```

Comment:
There is "solc -o OUTPUT_DIR" option which does great. That would also require splitting .sol files into files with names corresponding to contract names. This would decrease the size of our Makefile greatly and speed-up build process a lot.

I can prepare a separate PR with switch to it.